src/java/fr/paris/lutece/plugins/oauth2/jwt/JjwtJWTParser.java

```diff
@@ -84,12 +82,12 @@ public void parseJWT( Token token, AuthClientConf clientConfig, AuthServerConf s
             }
 
             JwtParser parser = parserBuilder.build( );
-            Claims claims;
+
             if ( serverConfig == null || serverConfig.getSignatureAlgorithmName( ) == null )
             {
                 // claims should be unsigned
-                Jwt<Header, Claims> jwt = parser.parse( strCompactJwt ).accept( Jwt.UNSECURED_CLAIMS );
-                claims = jwt.getPayload( );
+                Jwt<?, ?> jwt = parser.parse( strCompactJwt );
```

Comment:
why did you remove the `.accept( Jwt.UNSECURED_CLAIMS )` call ? It seems that this introduces a security risk by not enforcing assumptions anymore. If one does not yet cover this case, you should add a test that verifies that parsing fails if configured to parse unsigned tokens, but is passed signed one.